saga/sagalog.go

```diff
@@ -24,4 +24,12 @@ type SagaLog interface {
 	 * specified saga.
 	 */
 	GetMessages(sagaId string) ([]sagaMessage, error)
+
+	/*
+	 * Returns a list of all in progress sagaIds.
+	 * This MUST include all not completed sagaIds.
+	 * It may also included completed sagas
```

Comment:
Active Saga should be something that has started but not finished, but this is a perf optimization.  Implementations could just return everything and recovery will replay the log to figure out what's done vs not done.